### PR TITLE
Rlizzo/merge from upstream master

### PR DESCRIPTION
**Pull request by rlizzo**
Merged upstream master from https://github.com/iot-salzburg/gpu-jupyter

Resolves: GI-4369

### CODE REVIEW

File: .build/Dockerfile [hunk 1]
```diff
@@ -22,7 +22,8 @@ RUN chmod 1777 /tmp && chmod 1777 /var/tmp
 # Ubuntu 20.04 (focal)
 # https://hub.docker.com/_/ubuntu/?tab=tags&name=focal
 # OS/ARCH: linux/amd64
-ARG ROOT_CONTAINER=ubuntu:focal-20200423@sha256:238e696992ba9913d24cfc3727034985abd136e08ee3067982401acdc30cbf3f
+ARG ROOT_CONTAINER=ubuntu:focal-20210217@sha256:e3d7ff9efd8431d9ef39a144c45992df5502c995b9ba3c53ff70c5b52a848d9c
+
 
 LABEL maintainer="Jupyter Project <jupyter@googlegroups.com>"
 ARG NB_USER="jovyan"
```
Comment (luiscape) [inline .build/Dockerfile:28]: This looks like something we can change to, say, "gridai" for a nicer experience. Just to note.

File: .build/Dockerfile [hunk 2]
```diff
@@ -311,31 +344,33 @@ LABEL authors="Christoph Schranz <christoph.schranz@salzburgresearch.at>, Mathem
...
-RUN jupyter labextension install jupyterlab-plotly@4.8.1
-RUN jupyter labextension install @jupyter-widgets/jupyterlab-manager
-RUN pip install --no-cache-dir jupyter-tabnine==1.0.2  && \
-  jupyter nbextension install --py jupyter_tabnine && \
-  jupyter nbextension enable --py jupyter_tabnine && \
-  jupyter serverextension enable --py jupyter_tabnine
+
+# RUN jupyter labextension install @jupyterlab/github
+# RUN pip install jupyterlab-git
+RUN pip install jupyterlab-drawio
```
Comment (luiscape) [inline .build/Dockerfile:464]: Are all of these installed by default? Drawio and plotly?

**Review by luiscape — APPROVED** [review]
LGTM

File: .build/Dockerfile (hunk 1 above)
Comment (rlizzo) [reply .build/Dockerfile:28]: yeah, it's hardcoded in a few of the downstream scripts, but wouldn't be too much work to just find/replace all

File: .build/Dockerfile (hunk 2 above)
Comment (rlizzo) [reply .build/Dockerfile:464]: yes

**Pull request merged**